src/patternfly/components/Card/docs/code.md

```diff
@@ -1,12 +1,13 @@
 ## Overview
 
-A card is a generic rectangular container that can be used to build other components. A card can have three optional sections: header, body, and footer.
+A card is a generic rectangular container that can be used to build other components.
 
 ## Usage
 
-| Class                | Applied     | Outcome                                                                                                                                                                              |
-| -------------------- | ----------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------ |
-| `.pf-c-card`          | `<div>`     | Creates a card containing content.                                                                                                                                                     |
-| `.pf-c-card__header`          | `.pf-c-card`     | Creates the header of a card.                                                                                                                                                     |
-| `.pf-c-card__body`          | `.pf-c-card`     | Creates the body of a card.                                                                                                                                                 |
-| `.pf-c-card__footer`          | `.pf-c-card`     | Creates the footer of a card.                                                                                                                                                     |
+| Class | Applied | Outcome |
+| ---- | ---- | ---- |
+| `.pf-c-card` | `<div>` | Creates a card containing content. **Required** |
+| `.pf-c-card__header` | `.pf-c-card` | Creates the header of a card. |
+| `.pf-c-card__body` | `.pf-c-card` | Creates the body of a card. By default, the body element fills the available space in the card. You can use multiple `.pf-c-card__body` elements. |
+| `.pf-c-card__footer` | `.pf-c-card` | Creates the footer of a card. |
+| `.pf-m-no-fill` | `.pf-c-card__body` | Modifies one or more `.pf-c-card__body` elements to not fill the available space. |
```

Comment:
Could you change this to: Sets a `.pf-c-card__body` to not fill the available space in `.pf-c-card`. `.pf-m-no-fill` can be added to multiple card bodies.